stan/math/prim/mat/fun/LDLT_factor.hpp

```diff
@@ -94,8 +94,8 @@ namespace stan {
         return _ldltP->vectorD().array().log().sum();
       }
 
-      inline void inverse(Eigen::Matrix<T, R, C> &invA) const {
-        invA.setIdentity(N_);
+      template <typename MB> inline void inverse(Eigen::MatrixBase<MB> &invA) const {
```

Comment:
@jpritikin, it's failing cpplint.
Can you split this into two lines? (you're on your fork, which makes it harder for me to fix directly) the natural place is after the template argument.